tests/common/scripts/adapters/test_fhir_api.py

```diff
@@ -1,7 +1,73 @@
+import pytest
+
+from django.conf import settings
+
+import requests
+import responses
 from common.adapters.fhir_api import HapiFhirAPI
 
 
 def test_fhir_api():
     fhir_api = HapiFhirAPI()
 
-    assert fhir_api._headers == {"Cache-Control": "no-cache"}
+    assert fhir_api._headers == {"Cache-Control": "no-cache", "Content-Type": "application/fhir+json"}
+
+
+@responses.activate
```

Comment:
Why using `responses` instead of `InMemoryFhirAPI`?

Reply:
because I'm testing the `HapiFhirAPI` class, not `InMemoryFhirAPI`

Reply:
I renamed the file and test names